app/controllers/feedback_forms_controller.rb

```diff
@@ -1,37 +1,39 @@
 # frozen_string_literal: true
 
 class FeedbackFormsController < ApplicationController
-  skip_authorization_check
+  skip_authorization_check  # anyone can submit feedback
 
   def new; end
 
   def create
-    if request.post?
-      if validate
-        FeedbackMailer.submit_feedback(params, request.remote_ip).deliver_now
-        flash[:success] = t('blacklight.feedback_form.success')
+    return unless request.post?
+
+    if validate
+      FeedbackMailer.submit_feedback(params, request.remote_ip).deliver_now
+      flash[:success] = t('library-hours.feedback_form.success')
+    end
+
+    respond_to do |format|
+      format.json do
+        render json: flash
       end
-      respond_to do |format|
-        format.json do
-          render json: flash
-        end
-        format.html do
-          redirect_to params[:url]
-        end
+      format.html do
+        redirect_to params[:url]
       end
     end
   end
 
   protected
 
   def url_regex
-    /.*href=.*|.*url=.*|.*http:\/\/.*|.*https:\/\/.*/i
+    %r/.*href=.*|.*url=.*|.*https?:\/{2}.*/i
```

Comment:
I think you could change the enclosing characters to `()` and avoid having the escape the forward slash:
`%r(.*href=.*|.*url=.*|.*https?:/{2}.*)i`

Reply:
this was just a rubocop autocorrect. i tried with parenthesis and three tests failed (??) so i'm gonna leave it alone for now

Reply:
👍